R/bounds.r

```diff
@@ -1,3 +1,14 @@
+# If needed, converts input from classes that lack division and multiplication
+# to classes that can be scaled.
+# @param x vector of values to convert, if needed
+adapt_input_class <- function(x) {
```

Comment:
Using an S3 generic is the best approach, my apologies for not thinking about them when I wrote this patch. I was now able to provide a method (and test case) for `integer64` data as well.